lib/logstash/outputs/tcp.rb

```diff
@@ -38,6 +38,9 @@ class LogStash::Outputs::Tcp < LogStash::Outputs::Base
   # event will be written as a single line.
   config :message_format, :validate => :string, :deprecated => true
 
+  # respond_to? check needed for backwards compatibility with < 2.2 Logstashes
+  declare_workers_not_supported! if self.respond_to?(:workers_not_supported!)
```

Comment:
@jsvd nope! `workers_not_supported' is the legacy instance method. That's why we call both in this plugin. In v3.0.0 we can remove the legacy calls.